tests/e2e/tests/e2e/OpenshiftConnector.spec.ts

```diff
@@ -66,15 +68,14 @@ suite('Openshift connector user story', async () => {
         }
       ];
 
-    await driverHelper.navigateToUrl(TestConstants.TS_SELENIUM_BASE_URL);
+    await await browserTabsUtil.navigateTo(TestConstants.TS_SELENIUM_BASE_URL);
     await loginPage.login();
     await testWorkspaceUtils.createWsFromDevFile(wsConfig);
   });
 
   test('Login into workspace and open plugin', async () => {
-    await driverHelper.reloadPage();
-    await dashboard.waitPage();
-    await driverHelper.navigateToUrl(workspacePrefixUrl + wsName);
+    await dashboard.openDashboard();
```

Comment:
Do you need to `openDashboard` if you are right after that redirecting to a specific URL?

Reply:
Yes, for old dashboard it worked, direct Url went to IDE where test waited start ws. For new dashboard after the direct Url of ws, we are on dashboard quite long time waiting to start ws.

Reply:
As from what @ScrewTSW has tried, our [factory test](https://github.com/eclipse/che/blob/master/tests/e2e/tests/e2e/FactoryUrl.spec.ts#L30) is passing without problems against new dashboard. So maybe it would be enough just to increase a timeout if your workspace is larger and therefore start takes more time (setting one variable when tests are run). But I understand your intention and if you want to leave it as it is, I'm OK with that.

![Screenshot_20210412_182442](https://user-images.githubusercontent.com/29278800/114515326-86e41f00-9c3c-11eb-8552-5414b04e6d81.png)